nomad/node_endpoint.go

```diff
@@ -232,7 +246,6 @@ func (n *Node) Deregister(args *structs.NodeDeregisterRequest, reply *structs.No
 	}
 
 	// Determine if there are any Vault accessors on the node
-	ws := memdb.NewWatchSet()
 	accessors, err := n.srv.State().VaultAccessorsByNode(ws, args.NodeID)
```

Comment:
@dadgar Can we use the snap we created earlier?

Reply:
Yeah you can switch it over